lib/cli/query.ts

```diff
@@ -301,15 +304,15 @@ function argvQueryPrefold(argv, implicitTimestampOps) {
 
   if (d_age && implicitTimestampOps) {
     var now = Date.now();
-    var target = parseInt(now / 1000) - timeCli.timespecToSeconds(d_age);
```

Comment:
Bug fix: `now` is a `number`, and so is `now / 1000`, so there is no need `parseInt` here.